resources/views/livewire/stream-list-archive.blade.php

```diff
@@ -1,5 +1,5 @@
 <div>
-    <x-page-header title="Archive">
+    <x-page-header title="Archive ({{ $streams->count() }})">
```

Comment:
This count only holds the count of the current page streams (pagination), not the count of all streams.

Reply:
Gah, my bad! I didn't have enough test streams to kick the pagination in so it didn't even cross my mind. I've committed a change for this!

Thanks @christophrumpel 👍